code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -154,7 +154,7 @@
 	enemy_jobs = list("AI", "Cyborg", "Security Officer","Detective","Head of Security", "Captain", "Chaplain")
 	required_enemies = list(2,2,1,1,1,1,1,0,0,0)
 	required_candidates = 4
-	weight = 4
+	weight = 3
```

Comment:
🤔

Reply:
>not a single cult round since that PR
Balancing is hard

Reply:
well obviously if 3 is nevercult and 4 is alwayscult the solution would be to `weight = 3.5`